projects/fsastorefrontlib/src/assets/translations/de/common.de.ts

```diff
@@ -164,6 +164,7 @@ export const fscommon = {
       contact: 'Kontakt',
       categories: 'Kategorien',
       searchBox: 'Suchen Sie einen Agent oder eine Versicherungsart',
+      backToList: '[DE] BACK TO LIST',
```

Comment:
I am not familiar with CSS classes, but do we have one which will automatically perform upper case when it's applied on div?
I would avoid direct uppercase translation

Reply:
Yes, we have, I'll change it.